admin/src/css/theme.less

```diff
@@ -92,3 +114,18 @@ ul {
     transform: rotate(360deg);
   }
 }
+.nav-tabs>li.active>a {
+	color: @nav-active-color;
+	&:hover {
+		color: @nav-active-color;
+	}
+	&:focus {
+		color: @nav-active-color;
+	}
+}
+.nav-tabs > li > a {
+  color: @medic-slate-color;
+}
+.mb-5 {
```

Comment:
This is not a good class name because it describes what it does instead of when to use it. If we decide to change the margin to 10px or make it padding of 5px you'll have to change the class name everywhere it's used rather than just updating the implementation.

The equivalent in JavaScript is...

```js
const five = 5; // BAD
const timeout = 5; // GOOD
```